cookbooks/omnitruck/recipes/default.rb

```diff
@@ -15,14 +15,16 @@
   action :upgrade
 end
 
-hab_package 'chef-es/omnitruck-app'
-hab_package 'chef-es/omnitruck-poller'
-hab_package 'chef-es/omnitruck-web'
-hab_package 'chef-es/omnitruck-web-proxy'
+packages = %w(omnitruck-app omnitruck-poller omnitruck-web omnitruck-web-proxy)
+
+packages.each do |pkg|
+  hab_package "chef-es/#{pkg}" do
+    version node['applications'][pkg]
```

Comment:
You'll need to set this attribute if you want the version pinnings to be promoted throughput the AURD as expected:
https://github.com/chef/es-package-router/blob/master/.delivery/build-cookbook/attributes/default.rb#L29